docs/source/faq.rst

```diff
@@ -27,7 +27,8 @@ Training
    Then set the ``use_cuda`` flag in the configuration to True for training on GPU (requires CUDA) or to False for training on CPU.
 
 - **Does Joey NMT support multi-GPU processing?**
-   At the current stage, the code does not support multi-GPU processing. Contributions welcome :)
+   ~~At the current stage, the code does not support multi-GPU processing. Contributions welcome :)~~
+   [UPDATE] In version 1.0.0, we integrated multi-gpu and half-precision support.
```

Comment:
We could change this to: Yes, since version 1.0 we support multi GPU. (Thanks! :D)